.github/workflows/interactions-publish-to-npm.yml

```
@@ -0,0 +1,42 @@
+name: Wasm publish to npm
```

Should we give both workflows different names to make it easier to tell them apart?

E.g. here

```suggestion
name: Wasm publish `iota_interaction_ts` to npm
```

and in `.github/workflows/wasm-publish-to-npm.yml` (can't make a suggestion below)

```raw
name: Wasm publish `identity_wasm` to npm
```